static/styles/left_sidebar.css

```diff
@@ -287,6 +287,10 @@ li.show-more-topics {
     }
 }
 
+div.emoji {
+    display: inline-block;
+}
```

Comment:
This CSS selector is way too wide in scope -- it'll apply to ALL emoji in the app, regardless of whether they are in this sidebar. While that may be what we want in some situations, `static/styles/zulip.css`

On theory is we should add this to the general CSS here:

```
.emoji {                   
    height: 25px;          
    width: 25px;           
    position: relative;    
    margin-top: -7px;      
    vertical-align: middle;
    top: 3px;              
```

But I think the actual bug here is that we should be using a `span`, not a `div` in the HTML for the element in question; the main difference between those two elements is that spans are `inline-block`.

Reply:
Thanks for your review. I will use `span` instead of `div` as suggested.